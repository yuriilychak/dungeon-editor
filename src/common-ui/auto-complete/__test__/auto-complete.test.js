import React from "react";
import {createMount} from "@material-ui/core/test-utils";

import AutoComplete from "../auto-complete";

describe("AutoComplete test", () => {
    it("Default snapshot", () => {
        const wrapper = createMount()(
            <AutoComplete
                label="test"
                placeholder="test"
                showEmpty
                suggestions={[{
                    item: "test"
                }]}
                defaultItem="test"
                onAddItem={jest.fn()}
                onSelectItem={jest.fn()}
                onClearItem={jest.fn}
            />
        );

        expect(wrapper.html()).toMatchSnapshot();
    });
});
