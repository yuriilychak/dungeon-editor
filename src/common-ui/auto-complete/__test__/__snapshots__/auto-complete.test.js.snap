// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AutoComplete test Default snapshot 1`] = `"<div class=\\"makeStyles-root-1\\"><div class=\\"makeStyles-container-2\\" role=\\"combobox\\" aria-expanded=\\"false\\" aria-haspopup=\\"listbox\\" aria-labelledby=\\"downshift-options-label\\"><div class=\\"makeStyles-root-3\\"><div class=\\"MuiFormControl-root MuiTextField-root\\"><label class=\\"MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiFormLabel-filled\\" data-shrink=\\"true\\" for=\\"downshift-options-input\\" id=\\"downshift-options-label\\">test</label><div class=\\"MuiInputBase-root MuiInput-root makeStyles-inputRoot-4 MuiInput-underline MuiInputBase-formControl MuiInput-formControl\\"><input aria-invalid=\\"false\\" autocomplete=\\"off\\" id=\\"downshift-options-input\\" placeholder=\\"test\\" type=\\"text\\" aria-autocomplete=\\"list\\" aria-labelledby=\\"downshift-options-label\\" class=\\"MuiInputBase-input MuiInput-input makeStyles-inputInput-5\\" value=\\"test\\"></div></div></div><div role=\\"listbox\\" aria-labelledby=\\"downshift-options-label\\" id=\\"downshift-options-menu\\"></div></div></div>"`;
